docs/guides/continuous-integration/introduction.mdx

```diff
@@ -454,6 +454,10 @@ to see why the commit information is unavailable.
 DEBUG=commit-info,cypress:server:record
 ```
 
+:::info
+Cypress versions less than 3.15.0 will need to manually provide the CI ENVs to correctly send their SCM to the Cloud
```

Comment:
This is not a valid Cypress version. I'm not sure what @emilyrohrbough was trying to convey in their original comment. I think she meant the version that this is released in? https://github.com/cypress-io/cypress/pull/28577#issuecomment-1871298150

Reply:
The latest Cypress `3.x` version released was [cypress@3.8.3](https://docs.cypress.io/guides/references/changelog#3-8-3) in Jan 2020, so I would not have thought it is necessary to draw attention to such an old and legacy version.

With the exception of the [Configuration (Legacy)](https://docs.cypress.io/guides/references/legacy-configuration) page and the [Migration Guide](https://docs.cypress.io/guides/references/migration-guide) it is more or less assumed that the user has at the minimum Cypress `10.x` if not Cypress `13.x` running.

Reply:
@joshuadsouza are you able to make these updates?